test/run

```diff
@@ -6,13 +6,16 @@
 # IMAGE_NAME specifies a name of the candidate image used for testing.
 # The image has to be available before this script is executed.
 #
+# EXPRESS_REVISION specifies which express.js branch or tag should be tested.
 
 THISDIR=$(dirname ${BASH_SOURCE[0]})
 
 source "${THISDIR}/test-lib.sh"
 
 test -n $IMAGE_NAME \
   -a -n $VERSION
+# Change when new LTS version comes out
+readonly EXPRESS_REVISION="${EXPRESS_REVISION:-4.17.1}"
```

Comment:
I'd suggest to use the latest released version, that can be retrieved by `npm show express version`.

Reply:
Replaced with `$(npm show express version)` as default, but keeping the `EXPRESS_REVISION` variable to allow overriding it.